### DISCUSSION TITLE
should we unionize

### DISCUSSION THREAD

**Discussion by adamgeo1** [JoeMamasHoe/YeungalsClass-Superhero#2] [General]
i think it would be pretty cool if we unionized.

**Comment by marcoisabella9** [comment]
NO. AS UR BOSS I COMMAND U TO NOT UNIONIZE.

**Reply by adamgeo1** [reply]
what you gonna do about it

**Reply by marcoisabella9** [reply]
You are banned from the battle pass

**Reply by adamgeo1** [reply]
NOOOOOOOOOOOO

**Reply by nathanielmathew47** [reply]
ADAM YOU AR FIRED!!!!! KICK ROCKS WITH NO SOCKS PAL!!!
